Lib/test/test_capi/test_number.py

```diff
@@ -0,0 +1,330 @@
+import itertools
+import operator
+import sys
+import unittest
+import warnings
+
+from test.support import cpython_only, import_helper
+
+_testbuffer = import_helper.import_module('_testbuffer')
```

Comment:
Written like that, the whole file is skipped if _testbuffer is missing.

I expected something like:

```py
try:
    from _testbuffer import ndarray
except ImportError:
    ndarray = None
```

Or see other tests used _testbuffer.